### PR TITLE
tests(ticdc): move integration into tests

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 
part of https://github.com/pingcap/ticdc/issues/2199

### What is changed and how it works?
move integration into tests.

### Check List 

Tests 

 - No code

Code changes

None

Side effects

None

Related changes

None

### Release note 

```release-note
None
```

### CODE REVIEW

File: .github/workflows/ticdc_integration_cron.yaml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 name: CDC Integration Tests Cron
 
 on:
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - 'dm/**'
```
Comment (hi-rustin) [inline .github/workflows/ticdc_integration_cron.yaml:8]:
```suggestion
```

Remove this before merge.

**Comment by hi-rustin** [comment]
/cc @amyangfei @ben1009 

Could you please take a look? Thanks! 💚 💙 💜 💛 ❤️

**Review by overvenus — APPROVED** [review]

**Review by amyangfei — APPROVED** [review]

**Pull request merged**